src/renderers/shared/reconciler/__tests__/ReactStatelessComponent-test.js

```diff
@@ -125,6 +125,22 @@ describe('ReactStatelessComponent', function() {
     );
   });
 
+  it('should warn when given a ref', function() {
+    spyOn(console, 'error');
+
+    var Parent = React.createClass({
+      render: function() {
+        return <StatelessComponent name="A" ref="stateless"/>;
+      },
+    });
+    ReactTestUtils.renderIntoDocument(<Parent/>);
+
+    expect(console.error.argsForCall.length).toBe(1);
+    expect(console.error.argsForCall[0][0]).toContain(
+      'Stateless function components cannot be given refs.'
```

Comment:
Donezo. Also added the ref name into the error: `(See ref "stateless" in StatelessComponent created by Parent)`